examples/alert/alert.html

```diff
@@ -79,13 +79,13 @@ <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
           </tr>
         </thead>
         <tbody>
-          <tr>
+          <tr data-test-id="alert-role">
             <th scope="row"><code>alert</code></th>
             <td></td>
             <td><code>div</code></td>
             <td>Identifies the element as the container where alert content will be added or updated.</td>
           </tr>
-          <tr>
+          <tr data-test-id="alert-aria-live">
             <td></td>
             <th scope="row"><code>aria-live=<q>assertive</q></code></th>
             <td>Implicit on <code>div</code></td>
```

Comment:
We do not need a test for implicit property; not sure that this row should even be in the table. Oddly with this example, the thing that needs to be tested, pressing the button triggers the alert, is not really documented in the table because it is not part of the aria markup.